TGC/events/BanksAndUniversities.txt

```diff
@@ -0,0 +1,406 @@
+                    has_building = infrastructure }
+
+        modifier = { factor = 0.3                                       #Bigger chance to upgrade rather than building new one
+                    has_building = university }
+
+        modifier = { factor = 0.8                                       #Increase chance if the AI is reasearching university tech        
+                    positivism = 1 }                                    
+        modifier = { factor = 0.8                                      
+                    functionalism = 1 }                            
+        modifier = { factor = 0.8                                       
+                    biologism = 1 }                              
+        modifier = { factor = 0.7                                       
+                    social_science = 1 }
+        modifier = { factor = 0.7                                       
+                    social_alienation = 1 }
+    }
+
+	option = {
+		name = "AIEVENT" 
+		university = 1
+        owner = { treasury = -10000 }
+
+		random_pop = {                                       #Paying workers
+			limit = { strata = poor	}				  
+			money = 500       		  	                     #I took this idea from  u/Bearhobag's tax refund script
+		}								                     #This way, the money does not go to waste and gets injected back into the economy
+		random_pop = {										 #This will also make the economy healthier.
+			limit = { strata = poor	}	
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = poor	}
+			money = 500
+		}
+		random_pop = {
+			limit = { strata = middle }
+			money = 1000
+		}
+		random_pop = {
+			limit = { strata = middle }
+			money = 1000
+		}
+		random_pop = {
+			limit = { strata = rich }
+			money = 1000
+		}
+		random_pop = {
+			limit = { strata = rich }
+			money = 1000
+		}
+	}
+}  
+
+## Event so the AI also builds banks
+province_event = {
+
+	id = 6201
+    title = "AIEVENT"
+	desc = "AIEVENT"
+
+	trigger = { 
+        owner = { money = 10000
+                  NOT = { economic_policy = laissez_faire }
+				  ai = yes
+				  administration_spending = 0.6
```

Comment:
Note that when checking for administration_spending the game will crash if it's done for a country that doesn't actually exist. Generally for safety reasons youd put an exists = yes check before doing this just to be absolutely sure.

It is also basically the only case where you'd ever use exists = yes funnily enough

Reply:
I'm not gonna comment this everytime you check for it, I'm sure you can figure that out on your own